client/src/components/Profiles.js

```diff
@@ -83,78 +90,194 @@ const Profiles = () => {
       (profile.phone_number && isMatch(profile.phone_number)) ||
       (profile.occupation && isMatch(profile.occupation))
   );
+
+  const getClassNames = status => {
+    let color;
+    if (status === "new") {
+      color = "new";
+    } else if (status === "active") {
+      color = "purples";
+    } else color = "reds";
+    return color;
+  };
+  const getProfilesGroups = profileId => {
+    let profileGroupsNames = [];
+    let filteredGroups = groups.filter(group => group.members.length > 0);
+    filteredGroups.map(group =>
+      group.members.filter(member => {
+        if (member === profileId) {
+          profileGroupsNames.push(group.group_name);
+        }
+        return profileGroupsNames;
+      })
+    );
+    return profileGroupsNames;
+  };
+
+  let totalNewProfiles = 0;
+  profiles.forEach(profile => {
+    if (profile.status === "new") totalNewProfiles++;
+  });
+  let totalActiveProfiles = 0;
+  profiles.forEach(profile => {
+    if (profile.status === "active") totalActiveProfiles++;
+  });
+  let totalInactiveProfiles = 0;
+  profiles.forEach(profile => {
+    if (profile.status === "inactive") totalInactiveProfiles++;
+  });
+  let totalVolunteerProfiles = 0;
+  profiles.forEach(profile => {
+    if (profile.type === "volunteer") totalVolunteerProfiles++;
+  });
```

Comment:
```suggestion
  let totalNewProfiles = 0;
  let totalActiveProfiles = 0;
  let totalInactiveProfiles = 0;
  let totalVolunteerProfiles = 0;
   
  profiles.forEach(profile => {
    if (profile.status === "new") totalNewProfiles++;
    if (profile.status === "active") totalActiveProfiles++;
    if (profile.status === "inactive") totalInactiveProfiles++;
    if (profile.type === "volunteer") totalVolunteerProfiles++;            
  });
```
We can do this with a single `forEach` which is a bit cleaner.